internal/nexodus/wg_peers.go

```diff
@@ -0,0 +1,200 @@
+package nexodus
```

Comment:
git seems to think you removed one file and added a new one, so it's not showing the changes as a diff. Do you remember how you did it?  Was it a `git mv`?

Reply:
I used git mv and still see it as a delete and add. Reverted the filename to make the review not insane.